esmvaltool/diag_scripts/shared/_io.py

```diff
@@ -0,0 +1,234 @@
+"""Convenience functions for writing netcdf files."""
+import fnmatch
+import logging
+import os
+
+import iris
+
+logger = logging.getLogger(__name__)
+
+VAR_KEYS = [
+    'long_name',
+    'standard_name',
+    'units',
+]
+NECESSARY_KEYS = VAR_KEYS + [
+    'dataset',
+    'filename',
+    'project',
+    'short_name',
+]
+
+
+def _add_standard_name_to_iris(standard_name, units, cube=None):
```

Comment:
There has been some discussion about this recently, and I believe that the conclusion was that we should not invent our own standard names, because then they are not 'standard'. If you compute some variable that is not standard, it is best to simply not use the standard_name attribute. If you believe some variable name should become a standard name, we haven issue describing the process of getting it recognized as such https://github.com/ESMValGroup/ESMValTool/issues/718.

Reply:
Ok, I will remove that! :+1: